contracts/vaults/Base.sol

```diff
@@ -12,6 +12,7 @@ import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Burnable.sol";
 import "../tokenlock/ITokenLockFactory.sol";
+import "../RewardController.sol";
 
 
 contract  Base is OwnableUpgradeable, ReentrancyGuardUpgradeable {
```

Comment:
remove one space before `Base`

Reply:
yeah but let's not ask ben to fix things in this PR that have nothing to do with this PR